ddtrace/contrib/openai/__init__.py

```diff
@@ -0,0 +1,222 @@
+"""
+The OpenAI integration instruments the OpenAI Python library to emit metrics,
+traces and logs (disabled by default) for requests made to the
+completions, chat completions and embeddings endpoints.
```

Comment:
```suggestion
completions, chat completions, and embeddings endpoints.
```
We're an Oxford Comma house